src/dataframes.jl

```diff
@@ -8,7 +8,8 @@ end
 # if it's an array of symbols, set the labels and return a Vector{Any} of columns
 function handle_dfs(df::AbstractDataFrame, d::KW, letter, syms::AbstractArray{Symbol})
     get!(d, :label, reshape(syms, 1, length(syms)))
-    Any[collect(df[s]) for s in syms]
+    ret = Any[collect(df[s]) for s in syms]
+    hcat(ret...)
```

Comment:
Oh ok I misread... then the better approach would be: `Any[collect(df[s]) for i=1:1, s in syms]`  Though I thought Plots should be able to handle a `Vector{Any}` properly.